internal/params/config.go

```diff
@@ -596,6 +598,8 @@ type ChainConfig struct {
 	// vote power feature  https://github.com/harmony-one/harmony/pull/4683
 	// if crosslink are not sent for an entire epoch signed and toSign will be 0 and 0. when that happen, next epoch there will no shard 1 validator elected in the committee.
 	HIP32Epoch *big.Int `json:"hip32-epoch,omitempty"`
+
+	ViewChangeSkipValidatorsSameAddressEpoch *big.Int `json:"view-change-skip-validators-same-address-epoch,omitempty"`
```

Comment:
can we use the same epoch hardfork name as in https://github.com/harmony-one/harmony/pull/4798 ?